spec/models/player_spec.rb

```diff
@@ -177,5 +178,16 @@
         expect(player.side_bias).to eq(-3)
       end
     end
+    context 'with cut rounds in the mix' do
+      before do
+        create_list(:pairing, 2, player1: player, side: :player1_is_corp)
```

Comment:
i don't think you need create_list.  you are not using the array returned here.  We may also be misusing it in a couple of other places, though, so don't feel bad.  :D

Reply:
no, ignore me. I was reading too quickly.